src/plyql-mysql-gateway.ts

```diff
@@ -137,9 +137,9 @@ export function plyqlMySQLGateway(port: number, context: Datum, timezone: Timezo
               if (v && v.start) v = v.start;
 
               if (v && v.toISOString) {
-                v = dateToSQL(v);
+                v = Timezone.formatDateWithTimezone(v, timezone);
```

Comment:
you can not quite replace `dateToSQL` by `formatDateWithTimezone` because formatDateWithTimezone will give you a ISO string which is not the same as a SQL date. I think you want: `moment.tz(d, timezone.toString()).format('YYYY-MM-DD HH:mm:ss');`